woocommerce/woocommerce-ios
# Removed the shimmering modifier from OwnerUpgradesView

Closes: #11611


## Description

- There was a problem with SwiftUI Shimmer package so to fix the issue we ~~had to use exact commit reference until they merge and tag the proper fix~~ removed the **shimmering** modifier from **OwnerUpgradesView**
- The shimmer modifier was producing bug in a way that views were being animated in a loop by moving from bottom to top and messing up the entire view layout
- We will keep an eye on the [SwiftUI-Shimmer](https://github.com/markiv/SwiftUI-Shimmer/) repo and when the correct commit is merged we will update the package management
- ~~The last commit on [SwiftUI-Shimmer](https://github.com/markiv/SwiftUI-Shimmer/) main branch still has a bug so I used the commit from [unmerged PR](https://github.com/markiv/SwiftUI-Shimmer/pull/20)~~

## Testing instructions

- Open IAP upgrade screen
- If the content is loaded fast and you do not see any redacted and shimmering content you should check that the close button (X) and navigation title are positioned correctly and fixed on the top.
- Check gifs below and compare how it was before and how it is now

## Screenshots


| Before      | After |
| ----------- | ----------- |
| ![Simulator Screen Recording - iPhone 15 Pro - 2024-01-12 at 10 11 39](https://github.com/woocommerce/woocommerce-ios/assets/6242034/a04f613d-0c30-48c4-83fe-77f53d9a0df3)      |      ![Simulator Screen Recording - iPhone 15 Pro - 2024-01-16 at 10 40 42](https://github.com/woocommerce/woocommerce-ios/assets/6242034/6c0695e4-4730-4a1f-86f8-fcbc0ccbd39f) |




---
